traitsui/wx/ui_panel.py

```diff
@@ -742,6 +769,10 @@ def add_items(self, content, panel, sizer):
         col = -1
         col_incr = 1
         self.label_flags = 0
+
+        self._label_enabled_whens = []
```

Comment:
looks like we should make the changes made in https://github.com/enthought/traitsui/pull/1549 to qt to wx as well.

Reply:
I've opened #1592